spharpy/spherical.py

```diff
@@ -271,9 +282,21 @@ def spherical_harmonic_basis(n_max, coords):
     ----------
     n_max : integer
         Spherical harmonic order
-    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
-        Coordinate object with sampling points for which the basis matrix is
+    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>` or `sp.SamplingSphere <spharpy:classes/spharpy.samplings.coordinates>`
```

Comment:
Here and at some other places, lines are too long - can you check flake8 again, please?